config/default.yml

```diff
@@ -55,12 +55,12 @@ Sorbet/FalseSigil:
 
 Sorbet/ForbidIncludeConstLiteral:
   Description: 'Forbids include of non-literal constants.'
-  Enabled: true
+  Enabled: false
```

Comment:
question: would it work if we just delete this line? or do we need to specifically set it to `false`

Reply:
I couldn't find any documentation regarding the default value. Looks like even built in cops always specify `Enabled:` config https://github.com/rubocop-hq/rubocop/blob/master/config/default.yml so it maybe a faulty config to not use set this property 🤷